tests/api/test_elasticsearch.py

```diff
@@ -26,18 +26,24 @@ def test_cannot_access_bulk(self, client):
         assert client.delete('/api/v2/search/type/index/_bulk').status_code == 404
 
     def test_limitted_post(self, client):
-        assert client.post('/api/v2/search/type').status_code == 400
-        assert client.post('/api/v2/search/type/').status_code == 400
-        assert client.post('/api/v2/search/type/id').status_code == 400
-        assert client.post('/api/v2/search/type/id/').status_code == 400
-        assert client.post('/api/v2/search/type/id/some/thing/else').status_code == 400
-        assert client.post('/api/v2/search/type/id/some/thing/else/').status_code == 400
-        assert client.post('/api/v2/search/type/id/_search').status_code == 400
-        assert client.post('/api/v2/search/type/id/_search/').status_code == 400
-        assert client.post('/api/v2/search/type/id/some/thing/else/_search').status_code == 400
-        assert client.post('/api/v2/search/type/id/some/thing/else/_search/').status_code == 400
-        assert client.post('/api/v2/search/type/id/some/thing/else/_count').status_code == 400
-        assert client.post('/api/v2/search/type/id/some/thing/else/_count/').status_code == 400
+        assert client.post('/api/v2/search/type').status_code == 403
```

Comment:
@chrisseto, I'm not really sure what you had in mind for "pseudo fuzz testing". Would it be similar to the existing tests or more like a random url generator?